package.json

```diff
@@ -11,7 +11,7 @@
     "lint": "turbo run lint",
     "format": "prettier --write .",
     "dev": "turbo run dev",
-    "build": "turbo run build --no-cache --force",
+    "build": "turbo run build",
     "publish": "lerna publish",
```

Comment:
Would you be up for a (separate) PR to move to lerna-lite? lerna depends on nx, installing this repo I watched my computer lose ~15GBs of space (though about 5GBs got freed up), and it looked like nx was the biggest offender (and you use turbo instead).

(Personally I'd like to use pnpm because I use pnpm for everything else so space savings, but it's also your project so not seriously suggesting that)

Reply:
Since we're only using a subset of Lerna's features, I think it's a good idea.